utils/src/main/kotlin/ArchiveUtils.kt

```diff
@@ -112,6 +118,35 @@ fun File.unpack7Zip(targetDirectory: File) {
  */
 fun File.unpackZip(targetDirectory: File) = ZipFile(this).unpack(targetDirectory)
 
+/**
+ * A list with file names that are expected to be contained in a Debian package archive file.
+ */
+private val DEBIAN_ARCHIVE_CONTENT = listOf("data.tar.xz", "control.tar.xz")
+
+/**
+ * Unpack the [File] assuming it is a Debian archive. A Debian archive is an ar archive that contains itself two tar
+ * files with metadata and the actual content of the package. The top-level archive is deflated into a temporary
+ * directory. Then the tar files are extracted to the provided [targetDirectory].
+ */
+fun File.unpackDeb(targetDirectory: File) {
+    val tempDir = createOrtTempDir("unpackDeb")
+
+    try {
+        ArArchiveInputStream(inputStream()).unpack(
+            tempDir,
+            { entry -> entry.isDirectory || File(entry.name).isAbsolute },
+            { entry -> (entry as ArArchiveEntry).mode }
+        )
+
+        DEBIAN_ARCHIVE_CONTENT.forEach { name ->
+            val file = tempDir.resolve(name)
+            file.unpack(targetDirectory)
+        }
+    } finally {
```

Comment:
Does the above loop create the dirs "$targetDirectory/data", "$targetDirectory/content"?

Reply:
No, it just deflates the two archives to `targetDirectory`. There should not be any overlapping content in them.

Reply:
I'd vote for creating the dirs to not make assumptions about whether it overlaps.

Reply:
Done.